config.ru

```diff
@@ -13,5 +17,14 @@ end
 Sidekiq.configure_client do |config|
   config.redis = redis_hash
 end
-require "./app"
-run App
+
+use Rack::Env
```

Comment:
this line is unnecessesary beceause it's already loaded by the block between lines `9 - 15`, please check it